tools/iceoryx_build_test.sh

```diff
@@ -300,3 +311,30 @@ then
     $WORKSPACE/tools/gcov/lcov_generate.sh $WORKSPACE genhtml #generate html
     echo ">>>>>> Report Generation complete <<<<<<"
 fi
+
+#====================================================================================================
+#==== Step : Doxygen PDF Generation =================================================================
```

Comment:
@dkroenke Should this be called from your Python script in #481?

Reply:
Yes right we do not need to iterate through all doxyfiles anymore.
To build all doxygen docs just call: `./tools/iceoryx_build_test.sh clean build-all doc`

Reply:
Is `./tools/iceoryx_build_test.sh clean doc` also possible? That would be really useful for #481

Reply:
Yes, there is no need to build iceoryx within the doxygen generation.

Reply:
Great, thanks for the re-work!